src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/WordSimilarityChecker.cs

```diff
@@ -3,12 +3,11 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Collections.Generic;
 using System.Diagnostics;
 
 namespace Roslyn.Utilities
 {
-    internal class WordSimilarityChecker
+    internal struct WordSimilarityChecker : IDisposable
```

Comment:
💡 Likely want either `[NonCopyable]` or `readonly`

Reply:
yup.  needs to be NonCopyable.